### DISCUSSION TITLE
MVT Layer and TileLayer

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5553] [Q&A] [answered]
Hi, I just wanted to ask for some advice on our use case. We would like to visualize a dataset that consists of a large image (e.g. using the TileLayer) with vectorized data overlaid (e.g. points of interest and regions using the MVT example ). Does it make sense to use the both TileLayer and MVT Layer separately to render the image/vector data or is it possible to use the MVT layer for both the image and vector data? Would embedding the image data into the MVT tiles give us greater flexibility in rendering - e.g. change the color of the channels in the hypothetical MVT image data?

We have some examples we adapted from deck.gl's documentation using the two layers on ObservableHQ:
* [TileLayer](https://observablehq.com/@cornhundred/deck-gl-moon-tillayer-example)
* [MVTLayer](https://observablehq.com/@cornhundred/deck-gl-mvt-layer-custom?collection=@cornhundred/deck-gl)

**Comment by kylebarron** [comment, marked as answer]
> is it possible to use the MVT layer for both the image and vector data?

No, MVT tiles can only store vector data. It's impossible to store an image in a vector tile.

**Reply by cornhundred** [reply]
Thanks!
